src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -115,10 +122,11 @@ const TrackPanelListItem: FunctionComponent<TrackPanelListItemProps> = (
         <button onClick={changeDrawerViewHandler}>
           <img src={ellipsis.icon.on} alt={`Go to ${track.label}`} />
         </button>
-        <EyeToggleButton
-          iconStatus={trackStatus}
+        <ToggleImageButton
+          buttonStatus={trackStatus}
           description={'enable/disable track'}
           onClick={toggleTrack}
+          imageFile={Eye}
```

Comment:
it isn't a file anymore; it has been transformed into a ReactComponent. `image` perhaps? or `icon`?

Reply:
Changed it to `image`.